test/tbb/test_task_arena.cpp

```diff
@@ -1668,11 +1668,14 @@ void StressTestMixFunctionality() {
                     if (curr_arena == arenas_pool.end()) break;
 
                     curr_arena->arena->execute([] () {
-                        tbb::parallel_for(tbb::blocked_range<std::size_t>(0, 10000), [] (tbb::blocked_range<std::size_t>&) {
+                        static tbb::affinity_partitioner aff;
+                        auto body = [](tbb::blocked_range<std::size_t>&) {
                             std::atomic<int> sum{};
                             // Make some work
-                            for (; sum < 10; ++sum) ;
-                        });
+                            for (; sum < 10; ++sum);
+                        };
+                        tbb::parallel_for(tbb::blocked_range<std::size_t>(0, 10000), body, tbb::auto_partitioner{});
+                        tbb::parallel_for(tbb::blocked_range<std::size_t>(0, 10000), body, aff);
```

Comment:
Add the use case with `static_partitioner` as well?

Reply:
Can you please describe the supposed reason? The affinity and static partitioners use the same mechanism of affinity tasks that we want to test. So, it seems there is no value to increase testing because it will test the same stuff.

Reply:
> The affinity and static partitioners use the same mechanism of affinity tasks that we want to test

Well, not exactly the same - it is a bit different, otherwise there won't be different partitioners. Moreover, consider if one day the mechanism diverges more in these partitioners while the tests remain intact. We won't notice there is a problem with management of the tasks in the new mechanism, because I don't believe someone will recall to change this test accordingly.